### ISSUE TITLE
Function raised to a power gives an error

### ISSUE THREAD

**Issue by wassgha** [augustt198/latex2sympy#23]
Hello, I have noticed a few features that are not implemented yet. I have solved most of them by pre-processing input from MathQuill. I use this code:

`string.replace('\\right', '') # right tag
		.replace('\\left', '') # left tag
		.replace("^x","^{(x)}") # powers
		.replace("\ ","") #Spaces`

However, one thing I couldn't solve without changing latex2sympy code is when a user inputs a function raised to a power (say sin^nx instead of (sinx)^n) latex2sympy crashes and gives `NoneType' object has no attribute 'additive'`. 
Also, I believe hyperbolic trig functions are not supported yet (might be a very simple addition)
Can you please correct this? Thank you in advance!

**Comment by augustt198** [comment]
Yeah, there are a bunch of non-essential commands like `\right` that have to be ignored.

I hadn't noticed that exponents are so broken (I'm sure they worked better at one point). I'll fix that ASAP.

Hyperbolic trig functions (sinh, cosh, tanh, arsinh, arcosh, artanh) are currently supported, maybe you're on a different version?

**Comment by augustt198** [comment]
The exponent problem is fixed:
https://github.com/augustt198/latex2sympy/commit/f569090ad174a9b1b3d2cba5481c65e93b599ce3
